tests/testthat/test-tidiers.R

```diff
@@ -0,0 +1,110 @@
+# glance ----
+
+object <- get_mmrm()
+
+test_that("glance returns expected values in summary in tibble form", {
+  result <- object |> glance()
```

Comment:
also here and below please follow conventions in package to not use pipe when not needed e.g. for examples or vignette entries
```suggestion
  result <- glance(object)
```

Reply:
fixed